### ISSUE TITLE
server freezing

### ISSUE THREAD

**Issue by idangozlan** [malijs/mali#92]
I'm using malijs + gRPC for a long time without any issues. 
Recently, with growing traffic, I start facing a weird issue which the gRPC server is not responsive randomly and the only way to return it to life is to restart.

My first middleware is logging every request, and it's not logging any request when it's freezing. 

The way it looks from the client side is a stuck process waiting for the response.

No error / any indication is shown.

Wonder if someone had this issue?

```
"grpc": "1.19.0",
"mali": "0.14.0",
node carbon:alpine docker
```

**Comment by idangozlan** [comment]
Unary, Don't know what you mean about kind of traffic :), this is happening once a day in a random timing, since 7 days ago +-, grpc 1.19.0, it's really weird, I'm logging any kind of errors but it's not there. I considering remove malijs just to verify that it's not related, hopefully, I'll find it soon and will share here the case.

**Comment by idangozlan** [comment]
Seems like grpc has issue of memory and race condition which was fixed on v1.20.0.
